db/redis/storage/redis_test.go

```diff
@@ -11,6 +11,8 @@ import (
 	"sync"
 	"testing"
 	"time"
+
+	"github.com/NYTimes/video-transcoding-api/db"
```

Comment:
A quick note that this package doesn't depend on anything else in the video-transcoding-api on purpose: it's an independent library and if we want to extract this as an external repo, we could, and then users wouldn't need to download the video-transcoding-api code.

You tests don't change this because you're importing code from video-transcoding-api only in the test file, but the ideal approach would be to add a []struct{} field to some of the types declared in ``stub_test.go`` and use those types in the tests for FieldMap.